packages/pagination/Pagination.js

```diff
@@ -0,0 +1,108 @@
+import React, { useState, useContext } from 'react';
+import PropTypes from 'prop-types';
+import isFunction from 'lodash/isFunction';
+import sortBy from 'lodash.sortby';
```

Comment:
you are using components form the main lodash lib above (`lodash/isFunction`), but here you install and use the specific lib (`lodash.sortby`). Should be consistent.